### PR TITLE
Default to ES modules in GOV.UK Prototype Kit config

### PR DESCRIPTION
**Pull request by colinrotherham**
Completes the GOV.UK Frontend tasks from:

* https://github.com/alphagov/govuk-frontend/issues/3972

### CODE REVIEW

**Review by romaricpascal — COMMENTED** [review]
Tested locally with a fresh Prototype Kit and it works nicely 🥳 I'd be keen to see if we can avoid using paths that are decided on the Prototype Kit side to link our files, though 😊

**Comment by colinrotherham** [comment]
>Tested locally with a fresh Prototype Kit and it works nicely 🥳 I'd be keen to see if we can avoid using paths that are decided on the Prototype Kit side to link our files, though 😊

Thanks @romaricpascal, all that hard work paid off

File: packages/govuk-frontend/src/govuk-prototype-kit/init.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 // @ts-nocheck
+import { initAll } from '/plugin-assets/govuk-frontend/dist/govuk/govuk-frontend.min.js'
```
[the remark this answers is not in the document]
Comment (colinrotherham) [reply packages/govuk-frontend/src/govuk-prototype-kit/init.js]: @romaricpascal Yeah works great still, I've pushed it up 👍

For context, script paths are output in [**includes/scripts.njk**](https://github.com/alphagov/govuk-prototype-kit/blob/de8ccd949bb8fc4656568dd01779e2f12c726a57/lib/nunjucks/govuk-prototype-kit/includes/scripts.njk) for prototype pages

But you'll see errors in [**lib/nunjucks/views/manage-prototype/scripts.njk**](https://github.com/alphagov/govuk-prototype-kit/blob/de8ccd949bb8fc4656568dd01779e2f12c726a57/lib/nunjucks/views/manage-prototype/scripts.njk) as that's still in progress

**Review by romaricpascal — APPROVED** [review]
Glad the relative import worked! Thanks for trying it out and updating 🥳

**Pull request merged**